src/transformers/models/qwen2_audio/modeling_qwen2_audio.py

```diff
@@ -851,7 +776,7 @@ def forward(self, audio_features):
 class Qwen2AudioForConditionalGeneration(Qwen2AudioPreTrainedModel, GenerationMixin):
     def __init__(self, config: Qwen2AudioConfig):
         super().__init__(config)
-        self.audio_tower = AutoModel.from_config(config.audio_config)
+        self.audio_tower = AutoModel.from_config(config.audio_config)  # Usually a `Qwen2AudioEncoder` instance
```

Comment:
There were no references of `Qwen2AudioEncoder` usage in our codebase nor on the hub. However, upon closer inspection of checkpoints, we can see it is used here. Added a comment for clarification.

(btw, `Qwen2AudioEncoder` is untested 😢 )